tests/bsim/bluetooth/host/misc/disable/src/main_disable.c

```diff
@@ -35,6 +35,55 @@ static void test_disable_main(void)
 	PASS("Disable test passed\n");
 }
 
+static void test_disable_set_default_id(void)
+{
```

Comment:
```suggestion
{
	/* Provide some idle time for boot sequence to complete.
	 * Otherwise, it is observed that nRF POWER_CLOCK ISR does not get serviced.
	 */
	 k_sleep(K_MSEC(1));
 
```

Reply:
@aescolar any comments, why some test case work without this hack?

Reply:
Not really. I would need to dig into why that is happening, but I don't have time now. It would be nice if you or Emil did.

Reply:
@Thalley you are facing controller assertions, is similar workaround needed for your tests too?

Reply:
No, I haven't seen it for the bt_disable tests I'm implementing right now

Reply:
I see you already have couple `k_sleep()` calls that would allow ISRs to be vectored into.

Reply:
I added a temporary workaround. https://github.com/zephyrproject-rtos/zephyr/pull/73342 fixes the bug in the controller